WebAutopark/Repositories/SparePartsRepository.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Dapper;
+using Microsoft.Data.SqlClient;
+using System;
+using System.Data;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using WebAutopark.Interfaces;
+
+namespace WebAutopark.Repositories
+{
+    public class SparePartsRepository : IRepository<SparePart>
+    {
+        string connectionString = null;
+        public SparePartsRepository(string conn)
+        {
+            connectionString = conn;
+        }
+
+        public void Create(SparePart instance)
+        {
+            using (IDbConnection db = new SqlConnection(connectionString))
+            {
+                var sqlQuery = "INSERT INTO SpareParts (PartName) VALUES(@PartName)";
+                db.Execute(sqlQuery, instance);
+            }
+        }
```

Comment:
Move all connection creation logic into separate class, like BaseRepository, and inherits it.

Reply:
![image](https://user-images.githubusercontent.com/79978383/118659982-72281780-b7f6-11eb-9bb0-83898ef89c60.png)
Done